src/user.js

```diff
@@ -1,38 +1,30 @@
-const Pantry = require('../src/Pantry');
-const Recipe = require('../src/Recipe');
+const Pantry = require('../src/pantry');
+// const Recipe = require('../src/recipe');
 
 class User {
   constructor(obj) {
     this.name = obj.name;
     this.id = obj.id;
-    this.pantry = new Pantry(obj.pantry)
+    this.pantry = new Pantry(obj.pantry);
     this.favorites = [];
-    this.mealToCook = []
   }
 
-  addToFavorites() {
+  addToFavorites(recipe) {
     //adds recipe to the favorites array
+    this.favorites.push(recipe)
 
   }
 
   removeFromFavorites() {
-    // removes recipe from favorites array
-  }
-
-
-  stageMealToCook() {
-    //take current meal selection
-    //pass it into pantry.validateMeal
-    //^ if above equates to true, move meal to meal to cook on user
-    //^ if aboce equates to false. run pantry required for meal and return required items.
-    //add meal user wants to cook to meal que.
-  }
+    // thinking we can validate if it pushes or takes out 
+    // within the add to favorites method
 
-  cookMeal() {
```

Comment:
So I think cook meal maybe should live on the Pantry class. Thats how I have it set up after our discussion with Ryan the other day. How do you feel about that?